.github/workflows/continuous-integration.yml

```diff
@@ -105,16 +66,22 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -r requirements.txt
+          # All dependencies but markdown-it-py
+          python -m pip install nbformat pyyaml toml
           python -m pip install -r requirements-dev.txt
-          # install sphinx_gallery and matplotlib if available (may not work on pypy)
+          # install sphinx_gallery and matplotlib if available
           python -m pip install sphinx_gallery~=0.7.0 || true
           python -m pip install jupyter-fs || true
+      - name: Install markdown-it-py
+        if: ${{ matrix.markdown-it-py-version }}
+        run: python -m pip install markdown-it-py==${{ matrix.markdown-it-py-version }}
       - name: Install from source (required for the pre-commit tests)
```

Comment:
Not really a pre-commit here

Reply:
Yes sorry the comment is not very explicit - I mean the tests like https://github.com/mwouts/jupytext/blob/main/tests/test_pre_commit_0_ipynb_to_py.py

Reply:
Not a good naming if you ask me. Unless they are enforced/installable as pre-commit hooks.

Reply:
:smile: I do agree that names are very important.

What do you suggest to change? The comment, or the name of the test? In these tests we test Jupytext as a pre-commit hook, i.e. [.pre-commit-hooks.yaml](https://github.com/mwouts/jupytext/blob/main/.pre-commit-hooks.yaml)

Reply:
Ah, I understand now. Then the naming is good, but it should be a different job